rinja_parser/src/memchr_splitter.rs

```diff
@@ -7,7 +7,7 @@ pub(crate) trait Splitter: Copy {
     fn split<'a>(&self, haystack: &'a str) -> Option<(&'a str, &'a str)>;
 }
 
-impl<T: Splitter + ?Sized> Splitter for &T {
+impl<T: Splitter> Splitter for &T {
```

Comment:
Interesting, why didn't clippy like that? Because Splitter is not `pub`, so it's known that there won't be any unsized Splitters?

Reply:
Here's the output:

```
warning: `?Sized` bound is ignored because of a `Sized` requirement
  --> rinja_parser/src/memchr_splitter.rs:10:20
   |
10 | impl<T: Splitter + ?Sized> Splitter for &T {
   |                    ^^^^^^
   |
note: `T` cannot be unsized because of the bound
  --> rinja_parser/src/memchr_splitter.rs:10:9
   |
10 | impl<T: Splitter + ?Sized> Splitter for &T {
   |         ^^^^^^^^
   = note: ...because `Splitter` has the bound `Copy`
   = note: ...because `Copy` has the bound `Clone`
   = note: ...because `Clone` has the bound `Sized`
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#needless_maybe_sized
   = note: `#[warn(clippy::needless_maybe_sized)]` on by default
help: change the bounds that require `Sized`, or remove the `?Sized` bound
   |
10 - impl<T: Splitter + ?Sized> Splitter for &T {
10 + impl<T: Splitter> Splitter for &T {
   |
```